API_v1.md

```diff
@@ -123,39 +156,48 @@ Filter the widgets, one of:
 - **label** - widget label as currently displayed (i.e. translated!)
 - **type** - type of the widget
 
+Same as for reading, combinations of the filters are also allowed.
+
 Then specify the action:
 
 - **action** - the action to do
 - **value** (optional) - new value or a parameter of the action
 - **column** (optional) - column id when selecting item in the table
-
+- **row** (optional) - row id when selecting item in the table
 Supported actions:
 
 - **press** - to press the button
 - **check** | **uncheck**  | **toggle** - check, uncheck or toggle checkbox
 - **enter_text** - set text in the field, the text is passed in the
   *value* parameter
-- **switch_radio** - activate radio button
 - **select** - select value in the combobox, row in the table or node in the
   tree, requires *value*parameter
   - In case of table: select row in the table with given value. If
         *column* parameter is not provided, the first column will be used.  
   - In case of tree: select node in the tree. Use `|` as a delimiter for
         the child nodes.
 
-#### Response
+### Response
 
 JSON format
 
-#### Examples
+### Examples
 
 ```shell
 # press the "next" button
 curl -X POST 'http://localhost:9999/v1/widgets?id=next&action=press'
+# press the "next" button, filter by type and id
+curl -X POST 'http://localhost:9999/v1/widgets?type=YPushButtonid=next&action=press'
 # set value "test" for the InputField with label "Description"
 curl -X POST 'http://localhost:9999/v1/widgets?label=Description&action=enter_text&value=test'
+# select first row (counting from zero) in table with id "names"
+curl -X POST 'http://localhost:9999/v1/widgets?id=names&action=select&row=1'
 # select row with "test" cell value in the 2-nd column (counting from zero) in table with id "names"
 curl -X POST 'http://localhost:9999/v1/widgets?id=names&action=select&value=test&column=2'
 # select tree item with in tree with id "files"
 curl -X POST 'http://localhost:9999/v1/widgets?id=files&action=select&value=root|subnode|subnode'
-```
\ No newline at end of file
+# press url (<a href=\"firewall\">(enable)</a>) in richtext
+curl -X POST 'http://localhost:9999/v1/widgets?type=YRichText&action=select&value=firewall'
+# select menu item with label Image in parent item with label Document in YMenuButton menu
```

Comment:
SO we get `menubutton menu`? And which convention? I will put `menu button` if you think it helps with anything.

Reply:
I just saw a pattern in lower case in the other example, in case we want to follow it, that is not used the name starting with 'Y'.

Reply:
Yeah, for some cases it's clear what tree is, in case of menu button, I would say it's a bit more libyui specific. So yeah, I guess overall it's better now.